src/fights.ts

```diff
@@ -1957,3 +1958,32 @@ function voidMonster(): void {
   adventureMacro(determineDraggableZoneAndEnsureAccess(), Macro.basicCombat());
   postCombatActions();
 }
+
+export function printEmbezzlerLog(): void {
+  if (resetDailyPreference("garboEmbezzlerDate")) {
+    property.set("garboEmbezzlerCount", 0);
+    property.set("garboEmbezzlerSources", "");
+  }
+  const totalEmbezzlers =
+    property.getNumber("garboEmbezzlerCount", 0) +
```

Comment:
I don't think you need to use `getNumber` if you assign a default, but I could be wrong about this.